test/createApp.spec.js

```diff
@@ -16,4 +24,78 @@ describe('createApp', () => {
     expect(app).to.be.instanceof(App);
     expect(app.getOption('name')).to.eql('MyAppName');
   });
+
+
+  it('creates an instance with a validationFunctions option and returns it via getValidationFunctions()', () => {
+    const expectedValidationFunctions = ['expectedValue'];
+    const App = createApp({
+      appId: '123',
+      component: true,
+      name: 'MyAppName',
+      validationFunctions: expectedValidationFunctions,
+    });
+
+    const app = new App();
+
+    expect(app).to.be.instanceof(App);
```

Comment:
this expectation is not really what this test verifies, we can perfectly live without it.